static/src/stylesheets/module/content/_media.global.scss

```diff
@@ -15,8 +15,28 @@
         }
     }
 
+    .video__fader {
+        position: fixed;
+        z-index: 9999;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        background-color: rgba(0, 0, 0, .8);
+        opacity: 0;
+        transition: all 1s ease-in-out;
+        visibility: hidden;
+        
+        &.is-faded {
+            visibility: visible;
+            opacity: 1;
+            transition: all 1s ease-in-out;
+        }
+    }
+
     .player {
         margin-bottom: $gs-baseline/2;
+        z-index: 99999;
```

Comment:
This will inevitably give us grief in the future.
We've tried to add some [logic around applying `z-index`s](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/base/_zindex.scss) - it'd be good to see this used.